README.rst

```diff
@@ -23,5 +24,38 @@ change. The project was split out of the tests we wrote for our
 there.
 
 
+Optional integrations and testing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We have strong opinions about the testing tools we use, and we understand that
+other people may have equally strong opinions that differ from ours. For this
+reason, we have decided that none of seaworthy's core functionality will depend
```

Comment:
I've been writing it as `Seaworthy` in docs, while you've been doing `seaworthy`... We should probably agree on one.